### DISCUSSION TITLE
Is there anyway to disable 1.18.0s ability to be effect by the media keys

### DISCUSSION THREAD

**Discussion by GingerLandWhale** [pear-devs/pear-desktop#833] [Q&A]
I am in Uni at the moment and I find myself quite often watching muted tutorials while listen to music on your app ( which I love BTW, everything ive been looking for).
but when I go to pause and then play the tutorial again. it will pause the YT music app.
I have the pause and play inside the app set up on a shortcut.

Was just wondering if its not already in there. how hard would it be to do?
Thanks

**Comment by DerTyp7214** [comment]
Small tip, i don't know if it works on any os like this, but on windows you can control the secondary media sessions with your media keys while also pressing shift.

**Comment by QuditWolf** [comment]
![image](https://github.com/th-ch/youtube-music/assets/81188667/1b25e4f6-ade8-4a45-ac9c-cfb9beca0b6f)
Overriding Mediakeys looks to be it. worth a try. set the controls to something else etc.
If you're on linux you can set playerctl behaviour to affect(all-player, the first player, string patterns
